operations/integrations.md

```diff
@@ -194,6 +194,7 @@ Structure of command data varies depending on type of the [Device](integrations.
 | `Email` | string | optional | Email of the reservation owner. |
 | `KeyCount` | number | required | Count of keys to cut. |
 | `LockIds` | array of string | required | Identifiers of locks/rooms the key should open. |
+| `KeyMode` | string [KeyMode](integrations.md#keymode) | optional | Type of keys to encode. |
```

Comment:
Move closer together with KeyCount - whenever updating the docs, try to keep it "clean" as if it was completely created today. E.g. properties that are added later, but belong logically somewhere up should be up, not at the bottom.